### ISSUE TITLE
Unable to install requirements.txt modules

### ISSUE THREAD

**Issue by janvgils** [tomvdb/satellite-server#1]
Installing the solution on Debian amd64

```
virtualenv -p python3 virtenv
source ./virtenv/bin/activate

cd satellite-server
pip install -r requirements.txt
```

This will fail when python-apt and RPI_GPIO is part of the `requirements.txt` file, after removing both the installation went fine.

**Comment by tomvdb** [comment]
Yeah, the requirements were originally designed for the raspberry pi environment. I have cleaned up the requirements.txt a bit, but will do some more testing on different distro's and platforms.

Thanks!
